examples/chip-tool/commands/group/Commands.h

```diff
@@ -0,0 +1,372 @@
+
+        SetCommandExitStatus(CHIP_NO_ERROR);
+        return CHIP_NO_ERROR;
+    }
+
+private:
+    chip::GroupId groupId;
+    chip::KeysetId keysetId;
+};
+
+class UnbindKeySet : public CHIPCommand
+{
+public:
+    UnbindKeySet(CredentialIssuerCommands * credsIssuerConfig) : CHIPCommand("unbind-keyset", credsIssuerConfig)
+    {
+        AddArgument("groupId", chip::kUndefinedGroupId, UINT16_MAX, &groupId);
+        AddArgument("keysetId", 0, UINT16_MAX, &keysetId);
+    }
+    chip::System::Clock::Timeout GetWaitDuration() const override { return chip::System::Clock::Seconds16(5); }
+
+    CHIP_ERROR RunCommand() override
+    {
+        size_t index = 0;
+        chip::FabricIndex fabricIndex;
+        CurrentCommissioner().GetFabricIndex(&fabricIndex);
+        chip::Credentials::GroupDataProvider * groupDataProvider = chip::Credentials::GetGroupDataProvider();
+        auto iter                                                = groupDataProvider->IterateGroupKeys(fabricIndex);
+        size_t maxCount                                          = iter->Count();
+        chip::Credentials::GroupDataProvider::GroupKey groupKey;
+        while (iter->Next(groupKey))
+        {
+            if (groupKey.group_id == groupId && groupKey.keyset_id == keysetId)
+            {
+                break;
+            }
+            index++;
+        }
+        iter->Release();
+        if (index >= maxCount)
+        {
+            return CHIP_ERROR_INTERNAL;
+        }
+
+        ReturnErrorOnFailure(groupDataProvider->RemoveGroupKeyAt(fabricIndex, index));
+
+        SetCommandExitStatus(CHIP_NO_ERROR);
+        return CHIP_NO_ERROR;
+    }
+
+private:
+    chip::GroupId groupId;
+    chip::KeysetId keysetId;
+};
+
+class AddKeySet : public CHIPCommand
+{
+public:
+    AddKeySet(CredentialIssuerCommands * credsIssuerConfig) : CHIPCommand("add-keysets", credsIssuerConfig)
+    {
+        AddArgument("keysetId", 0, UINT16_MAX, &keysetId);
+        AddArgument("keyPolicy", 0, UINT16_MAX, &keyPolicy);
+        AddArgument("validityTime", 0, UINT64_MAX, &validityTime);
+        AddArgument("EpochKey", &epochKey);
+    }
+    chip::System::Clock::Timeout GetWaitDuration() const override { return chip::System::Clock::Seconds16(5); }
+
+    CHIP_ERROR RunCommand() override
+    {
+        chip::FabricIndex fabricIndex;
+        CurrentCommissioner().GetFabricIndex(&fabricIndex);
+        chip::Credentials::GroupDataProvider * groupDataProvider = chip::Credentials::GetGroupDataProvider();
+        uint8_t compressed_fabric_id[sizeof(uint64_t)];
+        chip::MutableByteSpan compressed_fabric_id_span(compressed_fabric_id);
+        ReturnLogErrorOnFailure(CurrentCommissioner().GetFabricInfo()->GetCompressedId(compressed_fabric_id_span));
+        // static const uint8_t epochKeySize = static_cast<uint8_t>(sizeof(chip::Credentials::GroupDataProvider::EpochKey));
```

Comment:
What is this commented-out bit doing here?

Reply:
removed